app/fissile_test.go

```diff
@@ -112,68 +112,68 @@ func TestListProperties(t *testing.T) {
 func TestDevDiffConfigurations(t *testing.T) {
 	assert := assert.New(t)
 	workDir, err := os.Getwd()
-	assert.Nil(err)
+	assert.NoError(err)
 
 	releasePathV215 := filepath.Join(workDir, "../test-assets/test-dev-config-diff/cf-release-215")
 	releasePathV224 := filepath.Join(workDir, "../test-assets/test-dev-config-diff/cf-release-224")
 	cachePath := filepath.Join(workDir, "../test-assets/test-dev-config-diff/cache")
 
 	release215, err := model.NewDevRelease(releasePathV215, "", "", cachePath)
-	if !assert.Nil(err) {
+	if !assert.NoError(err) {
 		return
 	}
 	assert.NotNil(release215)
 	release224, err := model.NewDevRelease(releasePathV224, "", "", cachePath)
-	assert.Nil(err)
+	assert.NoError(err)
 	assert.NotNil(release224)
 
-	assert.Equal(11, len(release215.Packages)) // temp #
-	assert.Equal(9, len(release224.Packages))
+	assert.Len(release215.Packages, 11) // temp #
```

Comment:
The comment (which was already there) is unclear.
Don't bother touching this, though :)